todo_app/accounts/tests.py

```diff
@@ -0,0 +1,84 @@
+from django.test import SimpleTestCase,  TestCase, RequestFactory
+from django.urls import reverse, resolve
+from rest_framework import status
+from django.contrib.auth.models import User
+from knox.models import AuthToken
+from rest_framework.test import APIClient, APIRequestFactory, force_authenticate
+from .api import RegisterAPI, LoginAPI, UserAPI
+from knox import views as knox_views
+
+
+# ==================================== TESTING URLS===========================================
```

Comment:
No need for the comments.
Also, it would be nice to have a `tests` directory, then separate files for what you're testing, so a `test_urls.py`, `test_views.py`, etc.